common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -77,37 +84,48 @@
             }
         ],
         "@typescript-eslint/consistent-type-definitions": [
-            "error"
+            "error",
+            "interface"
+        ],
+        "@typescript-eslint/consistent-type-imports": [
+            "warn",
+            {
+                "prefer": "type-imports"
+            }
         ],
         "@typescript-eslint/dot-notation": [
             "error"
         ],
-        "@typescript-eslint/explicit-function-return-type": [
+        "@typescript-eslint/eol-last": [
             "off"
         ],
+        "@typescript-eslint/explicit-function-return-type": [
+            "warn",
+            {
+                "allowConciseArrowFunctionExpressionsStartingWithVoid": false,
+                "allowDirectConstAssertionInArrowFunctions": true,
+                "allowExpressions": true,
+                "allowHigherOrderFunctions": true,
+                "allowTypedFunctionExpressions": true
+            }
+        ],
         "@typescript-eslint/explicit-member-accessibility": [
-            "off"
+            "error",
+            {
+                "accessibility": "explicit"
+            }
         ],
         "@typescript-eslint/explicit-module-boundary-types": [
-            "off"
-        ],
-        "@typescript-eslint/func-call-spacing": [
-            "off"
+            "error"
         ],
         "@typescript-eslint/indent": [
             "off"
         ],
-        "@typescript-eslint/interface-name-prefix": [
-            "off"
-        ],
-        "@typescript-eslint/keyword-spacing": [
```

Comment:
We should have keyword-spacing on (also thought we had this one on already on?)

Reply:
I have a perhaps unjustified dislike of formatting rules in lint. I'll prioritize getting tsfmt running. It's already mostly done (try `npm run tsfmt:fix` from the repo root).

I filed #9785 to track.

Reply:
In the meantime I'd like to avoid regressions.  Can you please leave these on until tsfmt is ready?